src/process-columnar-results.js

```diff
@@ -1,19 +1,26 @@
 /**
-   * Because it is inefficient for the server to return a row-based
-   * data structure, it is better to process the column-based results into a row-based
-   * format after the fact.
-   *
-   * @param {TRowSet} data The column-based data returned from a query
-   * @param {Boolean} eliminateNullRows A flag that allows removal of null rows from results
-   * @param {Object} dataEnum A list of types created from when executing {@link #invertDatumTypes}
-   * @returns {Object} processedResults The formatted results of the query
-   */
-export default function processColumnarResults (data, eliminateNullRows, dataEnum) {
-  const formattedResult = {fields: [], results: []}
+ * Because it is inefficient for the server to return a row-based
+ * data structure, it is better to process the column-based results into a row-based
+ * format after the fact.
+ *
+ * @param {TRowSet} data The column-based data returned from a query
+ * @param {Boolean} eliminateNullRows A flag that allows removal of null rows from results
+ * @param {Object} dataEnum A list of types created from when executing {@link #invertDatumTypes}
+ * @returns {Object} processedResults The formatted results of the query
+ */
+export default function processColumnarResults(
+  data,
+  eliminateNullRows,
+  dataEnum
+) {
+  const formattedResult = { fields: [], results: [] }
   const numCols = data.row_desc.length
-  const numRows = typeof data.columns[0] === "undefined" ? 0 : data.columns[0].nulls.length
+  const numRows =
+    typeof data.columns[0] === "undefined" ? 0 : data.columns[0].nulls.length
+  // to satisfy eslint no-magic-numbers rule
+  const oneThousand = 1000
```

Comment:
Same comment as above - also, with the logic in multiple places, could use an abstracted function as well.

Reply:
Or even a generic row formatter function setup, given that BOOL is also processed.

Reply:
Perhaps this is too large a change to consider for the formatting introduction, though. Though if so, should we capture?

Reply:
refactoring is nice, but let's do that in a separate PR.  in this case, I think `oneThousand` is more clear than `Milliseconds`

will change to `oneThousandMilliseconds` to be diplomatic